src/ngraph_builder.cc

```diff
@@ -1619,11 +1630,12 @@ static tf::Status TranslateSquareOp(const tf::Node* op,
 
 static tf::Status TranslateSquaredDifferenceOp(const tf::Node* op,
                                                Builder::OpMap& ng_op_map) {
-  return TranslateBinaryOp(op, ng_op_map, [](std::shared_ptr<ng::Node> input1,
```

Comment:
Is this reformatting intentional or caused by some different branch checkout?

Reply:
Reformatting due to clang.